rasa/core/migrate.py

```diff
@@ -274,14 +274,16 @@ def migrate_domain_format(
                 f"domain file. Only domain yaml files can be migrated. "
             )
 
-        backup = current_dir / "original_domain.yml"
-        original_domain = _create_back_up(domain_file, backup)
+        backup_dir = current_dir / "original_domain.yml"
```

Comment:
Almost there! 💯 
This could be nitpicking, but I'd rather rename all `occurences` of `backup_dir` to a more general name (`backup_location` or any other), otherwise having this assignment of file path to a name that contains `dir` is misleading and confusing for somebody looking in this file for first time.

Reply:
yes I was thinking the same 👍